media/js/firefox/mobile/mobile.js

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla, $) {
+    'use strict';
+
+    var $body = $('body');
+    var $mobileDownloadButtons = $('.mobile-download-buttons').remove();
+    var $modalContents = $('#modal-wrapper');
+
+    function openModal(e) {
+        e.preventDefault();
+
+        var product = $(this).data('product');
+
+        var selectorToHide = (product === 'firefox') ? '.focus' : '.firefox';
+        var selectorToShow = (product === 'firefox') ? '.firefox' : '.focus';
+
+        // contorl styling of modal: blue for firefox, purple for focus
+        $body.attr('data-modal-product', product);
+
+        $modalContents.find(selectorToHide).addClass('hidden');
+        $modalContents.find(selectorToShow).removeClass('hidden');
+
+        Mozilla.Modal.createModal(this, $modalContents, {
+            title: ''
```

Comment:
Is there a reason to pass an empty `title` here?

Reply:
Ah, nope. Will remove.